server/lib/config/pbench-server-default.cfg

```diff
@@ -114,12 +113,9 @@ host_info_url = http://%(webserver)s/pbench-results-host-info.versioned/pbench-r
 host = %(default-host)s
 user = %(default-user)s
 mailfrom = %(user)s@%(host)s
-shims = prep-shim-001 prep-shim-002
+shims = prep-shim-002
```

Comment:
Where is this used? I don't remember it at all.

Reply:
Found it - it's used by `pbench-server-activate-create-crontab`.

Reply:
Ok.. Thank you